### PR TITLE
743: Step 7 font updates

### PR DESCRIPTION
**Pull request by Mdemenko1**
## What (if any) features are you implementing?

Work for the issue [743](https://github.com/Gary-Community-Ventures/benefits-calculator/issues/743)

Updating fonts (old version) 

image

## Updated import

image

### CODE REVIEW

**Review by CalebPena — CHANGES_REQUESTED** [review]
It looks like the `Roboto Slab` font is not properly imported, so it is showing the wrong font on the header.

**Comment by Mdemenko1** [comment]
> It looks like the `Roboto Slab` font is not properly imported, so it is showing the wrong font on the header.

@sydney-devine Please confirm the font

@CalebPena I think this is exactly how it should look like, based on this discussion https://github.com/Gary-Community-Ventures/benefits-calculator/issues/743

**Comment by sydney-devine** [comment]
> > It looks like the `Roboto Slab` font is not properly imported, so it is showing the wrong font on the header.
> 
> @sydney-devine Please confirm the font
> 
> @CalebPena I think this is exactly how it should look like, based on this discussion #743

So, @CalebPena and @ivonne-hernandez were digging into this a little. It sounds like maybe the font wasn't properly imported. Robot Slab does indeed have the tails on the letters, but the tails should look a little blockier than what's showing in the screenshot above. 

Here's what it looks like based on the branding guide. 
image

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link
       rel="stylesheet"
-      href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700&family=Roboto:300,400,500,700&family=Roboto+Slab:700&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;700&family=Roboto:wght@300;400;500;700&family=Roboto+Slab:wght@700&display=swap"
```
Comment (CalebPena) [inline public/index.html:27]: The font still is not importing for me.
image
image

**Review by CalebPena — CHANGES_REQUESTED** [review]

**Review by CalebPena — APPROVED** [review]

**Pull request merged**